redirects-list.json

```diff
@@ -17,6 +17,14 @@
   "^https://error\\.dvc\\.org/(.*)?                                                       https://dvc.org/doc/user-guide/troubleshooting#$1 303",
   "^https://(code|data|remote)\\.dvc\\.org/(.+)                                           https://s3-us-east-2.amazonaws.com/dvc-public/$1/$2 303",
 
+  "^https://s\\.dvc\\.org/g/exp/run(.*)?$                                                https://dvc.org/doc/user-guide/experiment-management/running-experiments$1",
+  "^https://s\\.dvc\\.org/g/exp/(.*)?$                                                   https://dvc.org/doc/user-guide/experiment-management/$1",
+  "^https://s\\.dvc\\.org/g/pipeline-files/(.*)?$                                        https://dvc.org/doc/user-guide/project-structure/pipelines-files/$1",
+  "^https://s\\.dvc\\.org/g/(.*)?$                                                       https://dvc.org/doc/user-guide/$1",
+  "^https://s\\.dvc\\.org/s/(.*)?$                                                       https://dvc.org/doc/start/$1",
+  "^https://s\\.dvc\\.org/c/(.*)?$                                                       https://dvc.org/doc/use-cases/$1",
+  "^https://s\\.dvc\\.org/r/(.*)?$                                                       https://dvc.org/doc/command-reference/$1",
```

Comment:
If these are temporary, we should use `301` I think.

Reply:
I think it should be the same as we do for the short command reference links?

Reply:
I don't think these are temporary, but not permanent for HTTP 301. 

A general URL shortener solution seems another site to manage and I believe this one provides a good solution we already use. 

So, an HTTP 302 may be used as a status code.

Reply:
302 (temporary) sounds right to me... probably @jorgeorpinel also meant 302